navigator_engine/common/graph_visualizer.py

```diff
@@ -0,0 +1,209 @@
+import navigator_engine.model as model
+from dash import dcc
+import dash_cytoscape as cyto
+from dash import html
+from dash.dependencies import Input, Output
+
+
+graph_stylesheet = [  # Group selectors
```

Comment:
Please can the comment just go on the line above to be consistent with the other comments?